kyo-prelude/shared/src/main/scala/kyo2/kernel/Pending.scala

```diff
@@ -50,6 +51,26 @@ object `<`:
                         )
             mapLoop(v)
         end map
+
+        inline def evalNow: Maybe[A] =
```

Comment:
Could this be the new `pure`?

Reply:
I had named it `pure` initially but thought it could be confusing given that we used to provide the method with a different behavior. `evalNow` seems to better communicate what the API does as well.